src/__tests__/loadable-components/intersection-observer.test.js

```diff
@@ -104,4 +105,22 @@ describe('Loadable', () => {
 
     expect(globallyTrackedElements.length).toEqual(1)
   })
+
+
+  test('it should call setIntersectionObserver with observerOptions', () => {
+    const mock = jest.fn().mockImplementation(() => {
+      return {
+        observe: () => {},
+      }
+    })
+
+    global.IntersectionObserver = mock
+
+    const Loader = loadableVisiblity(opts)
+
+    mount(<Loader {...props} />)
+
+    expect(mock).toHaveBeenCalledWith(expect.anything(), opts.observerOptions)
+    global.IntersectionObserver = IntersectionObserver
```

Comment:
Ideally this and setting up the mock should be done in a `beforeEach` and `afterEach` for this test (perhaps wrap it in a `describe`), such that the mock is ensured to be cleaned up in the case that this test exits prematurely.